packaging/dependencies.nix

```diff
@@ -55,4 +82,6 @@ scope: {
       CONFIG_ASH_TEST y
     '';
   });
+
+  mkMesonDerivation = f: stdenv.mkDerivation (lib.extends localSourceLayer f);
```

Comment:
No meson-specific stuff here yet, but I'm sure we'll want to make use of it that way.